generator/src/main/java/com/scottlogic/deg/generator/violations/ManifestDTO.java

```diff
@@ -1,4 +1,4 @@
-package com.scottlogic.deg.generator.outputs.manifest;
+package com.scottlogic.deg.generator.violations;
```

Comment:
I would prefer if this type was either:
- kept in the `outputs` package, to highlight its scope
- moved to the `schemas` module, as it highlights its scope & introduces the ability for others to use it when de-serialising the manifest.

I'd prefer the latter if possible, but dont think moving it to the `violations` package is correct here.